packages/shared/src/components/utilities.tsx

```diff
@@ -118,7 +118,7 @@ export const NewCommentContainer = classed(
 
 export const ResponsivePageContainer = classed(
   PageContainer,
-  'py-6 laptop:border-theme-divider-tertiary laptop:border-l laptop:border-r laptop:min-h-screen',
+  'py-6 laptop:border-theme-divider-tertiary laptop:border-l laptop:border-r laptop:min-h-screen pb-[1.4375rem]',
```

Comment:
The original value was 23 which is 5.75rem, with that, I think we can round it off to just 24 to lessen arbitrary value. What do you think?
```suggestion
  'py-6 laptop:border-theme-divider-tertiary laptop:border-l laptop:border-r laptop:min-h-screen pb-24',
```

Reply:
Ah good point, took it very literal, but yeah think we can do this rounding.